run_mypy_and_annotate.py

```diff
@@ -0,0 +1,37 @@
+import re
+import subprocess
+
+
+def run_mypy():
```

Comment:
here is the script where we annotate offending lines with `# type: ignore`! i think it is worth having someone review the script, though maybe we don't want to actually commit it to the repo

Reply:
oh cool, i didn't know that repo existed. definitely seems like this belongs there!